hack/lib/serverless.bash

```diff
@@ -152,22 +152,24 @@ function deploy_knativeserving_cr {
   # Deploy the full version of KnativeServing (vs. minimal KnativeServing). The future releases should
   # ensure compatibility with this resource and its spec in the current format.
   # This is a way to test backwards compatibility of the product with the older full-blown configuration.
-  oc apply -n "${SERVING_NAMESPACE}" -f "${rootdir}/test/v1alpha1/resources/operator.knative.dev_v1alpha1_knativeserving_cr.yaml"
-
   if [[ $FULL_MESH == "true" ]]; then
-    enable_net_istio
+    deploy_with_istio
+  else
+    oc apply -n "${SERVING_NAMESPACE}" "${rootdir}/test/v1alpha1/resources/operator.knative.dev_v1alpha1_knativeserving_cr.yaml"
```

Comment:
Maybe add a comment here to show that we are installing kourier, since there is `no deploy_with_kourier` func, or maybe add one?

Reply:
OK. Makes sense.